index.html

```diff
@@ -465,19 +465,26 @@ <h4 id="is-response-eligible-for-integrity-validation">Is <var>response</var> el
       <p>The following algorithm details these restrictions:</p>
 
       <ol>
-        <li>Let <var>request</var> be the request that fetched
-<var>resource</var>.</li>
-        <li>If the <a href="https://fetch.spec.whatwg.org/#concept-response-type">response type</a> is <code>opaque</code>, return <code>false</code>.</li>
-        <li>If the <a href="https://fetch.spec.whatwg.org/#concept-request-mode">mode</a> of <var>request</var> is <code>CORS</code>,
-return <code>true</code>.</li>
-        <li>If the <a href="https://fetch.spec.whatwg.org/#concept-request-origin">origin</a> of <var>request</var> is
-<var>resource</var>’s origin, return <code>true</code>.</li>
+        <li>Let <var>response</var> be the response that results from
+<a href="https://fetch.spec.whatwg.org/#main-fetch">fetching</a> the <var>resource</var>.</li>
+        <li>If the <var>response</var> <a href="https://fetch.spec.whatwg.org/#concept-response-type">type</a> is <code>basic</code>,
+<code>cors</code> or <code>default</code>, return <code>true</code>.</li>
         <li>Return <code>false</code>.</li>
       </ol>
 
-      <p class="note">Step 3 returns <code>true</code> if the fetch was a CORS-enabled request. If the
-fetch failed the CORS checks, it won’t be available to us for integrity
-checking because it won’t have loaded successfully.</p>
+      <div class="note">
+        <p>The <a href="https://fetch.spec.whatwg.org/#concept-response-type">response types</a> are defined by the Fetch
+specification [[!FETCH]], and refer to the following:</p>
+
+        <ul>
+          <li><code>basic</code> is a same-origin response, and thus the requestor has full access
+to read the content.</li>
+          <li><code>cors</code> is a valid response to a cross-origin, CORS-enabled request, and thus
+again the requestor has full access to read the content.</li>
```

Comment:
I guess this is fine, but you don't get all content. Headers are safelisted. But even "basic" has `Cookie` headers blocklisted.